lib/repl.js

```diff
@@ -289,6 +289,10 @@ function REPLServer(prompt,
     debug('domain error');
     const top = replMap.get(self);
     internalUtil.decorateErrorStack(e);
+    if (e.stack && self.replMode === exports.REPL_MODE_STRICT) {
+      e.stack = e.stack.replace(/(\s+at\s+repl:)(\d+)/,
+                                (_, pre, line) => pre + (parseInt(line) - 1));
```

Comment:
@Fishrock123 `parseInt` is removed. parenthesis around subtraction is required because arithmetic operators are left associative and `pre` is a string.